.github/workflows/maven.yml

```diff
@@ -286,6 +300,11 @@ jobs:
           if [[ ${{matrix.benchmark}} == templated ]]; then
             java -jar benchbase.jar -b tpcc -c config/mysql/sample_tpcc_config.xml --create=true --load=true --execute=false --json-histograms results/histograms.json
             java -jar benchbase.jar -b ${{matrix.benchmark}} -c config/mysql/sample_${{matrix.benchmark}}_config.xml --create=false --load=false --execute=true --json-histograms results/histograms.json
+          # For anonymization, we load tpcc and anonymize a single table. The workload itself is not executed
+          elif [[ ${{matrix.benchmark}} == anonymization ]]; then
+            java -jar benchbase.jar -b tpcc -c config/mysql/sample_tpcc_config.xml --create=true --load=true --execute=false --json-histograms results/histograms.json
+            java -jar benchbase.jar -b tpcc -c config/mysql/sample_${{matrix.benchmark}}_config.xml --anonymize=true
+            exit 0
```

Comment:
```suggestion
```

Reply:
necessary as long as no actual benchmark is run (otherwise it will always fail because it does not find any results of benchmarking runs

Reply:
Understood.  I guess this goes along with the explanation in the other comment about not executing yet.  Please put a comment in for now so we know to remove this part once the execution phase is added.